packages/console/app/assets/css/entities.css

```
@@ -114,33 +114,63 @@
 	background-color: color-mix(in oklab, var(--flag) 18%, transparent);
 }
+
+/* Pending add: the span the reviewer is about to add, while the popover is open.
+   Reads as a text selection — a solid fill, no underline — so it clearly marks
+   what's selected once the native (blue) selection is cleared. A selection-blue
+   tint (the palette is otherwise monochrome) matches the browser convention and
+   stays legible on both themes. Overrides the underline chip treatment above. */
+.chip[data-category="pending"],
+.docx-chip[data-category="pending"] {
+	--select: oklch(0.62 0.16 255);
+	box-shadow: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)